### PR TITLE
Use `GetAttribute` to enable Draco quantization when possible

### PR DESCRIPTION
**Pull request by ebogo1**
This is a temporary workaround for https://github.com/CesiumGS/cesium/issues/9847. See https://github.com/google/draco/issues/742#issuecomment-954934088 for more info. Since `SkipAttributeTransform` breaks lookups with `GetAttributeByUniqueId`, this PR uses the alternative approach with `GetAttributeId` and `GetAttribute` to preserve quantization when possible. 

The downside is that this might not work when there are multiple sets of an attribute, i.e. `TEXCOORD_0` and `TEXCOORD_1`. Because of this, we only call `SkipAttributeTransform` with the new `GetAttribute` approach for "single" attributes. Otherwise, we continue without quantization and use the `GetAttributeByUniqueId` call as before.

I'm working on adding back some of the specs that were removed previously, but the core change is all in decodeDraco.js.

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @ebogo1!
* :grey_exclamation: There was an error checking the CLA! If this is your first contribution, please send in a [Contributor License Agreement](https://github.com/CesiumGS/cesium/blob/main/CONTRIBUTING.md#contributor-license-agreement-cla).
    * Maintainers, this was the error I ran into while attempting to process the CLA check. Please resolve it to continue CLA checking.
    * You'll need to [manually check for a submitted CLA](https://github.com/CesiumGS/cesium/blob/main/Documentation/Contributors/CLAs/README.md)
    ```
    Error: The service is currently unavailable.
    ```
* :grey_question: [CHANGES.md](https://github.com/CesiumGS/cesium/blob/draco-workaround/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.
* :grey_question: Unit tests were not updated.
    * Make sure you've [updated tests](https://github.com/CesiumGS/cesium/tree/main/Documentation/Contributors/TestingGuide) to reflect your changes, added tests for any new code, and ran the code coverage tool.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.

File: Source/WorkersES6/decodeDraco.js [hunk 1]
```diff
@@ -275,6 +275,24 @@ function decodePointCloud(parameters) {
...
+  // calling SkipAttributeTransform(), we will not skip attributes with multiple
+  // sets of data in the glTF.
+  var attributesToSkip = ["POSITION", "NORMAL"];
+  var compressedAttributes = parameters.compressedAttributes;
+  if (!defined(compressedAttributes["COLOR_1"])) {
+    attributesToSkip.push("COLOR");
+  }
+  if (!defined(compressedAttributes["TEXCOORD_1"])) {
+    attributesToSkip.push("TEX_COORD");
+  }
```
Comment (lilleyse) [inline Source/WorkersES6/decodeDraco.js:289]: glTF does allow for `TEXCOORD_2`, `TEXCOORD_3`, etc so some of this glTF attribute name -> draco name mapping could be organized in a more general way. Though to be fair, `TEXCOORD_2` and higher are only used in `ModelExperimental` and if we're expecting a bug fix in draco soon then maybe we don't have to worry about it here.

Comment (ebogo1) [reply Source/WorkersES6/decodeDraco.js:289]: My thinking was that if there were more than one TEXCOORD, they numbering would go in order, meaning that at least `TEXCOORD_1` would exist - in which case, we would not call `SkipAttributeTransform()`. Is it possible to just have one TEXCOORD and call it `TEXCOORD_2`? I'm not sure I'm following your comment.

Comment (lilleyse) [reply Source/WorkersES6/decodeDraco.js:289]: Oh that's true, never mind then. From the glTF spec:

> All indices for indexed attribute semantics MUST start with 0 and be consecutive positive integers

**Comment by lilleyse** [comment]
Thanks for the quick turnaround @ebogo1. For 99% of cases we can use quantization again!

**Comment by lilleyse** [comment]
Forgot to mention, I tested all the 3D models and 3D Tiles related sandcastles and ran the unit tests locally.

**Comment by ebogo1** [comment]
Thanks @lilleyse. I tested [microcosm-draco.gltf.zip](https://github.com/CesiumGS/cesium/files/7445349/microcosm-draco.gltf.zip) (converted from https://github.com/CesiumGS/cesium/tree/main/Specs/Data/Models/GltfLoader/Microcosm/glTF with gltf-pipeline), which has multiple TEXCOORD attributes and all seems to work as expected.

**Pull request merged**